scripts/dev/run_memcheck.in

```diff
@@ -7,4 +7,12 @@ fi
 
 ARGS="-j@PROCESSOR_COUNT@ --force-new-ctest-process -T memcheck --build-config $1 --output-on-failure -LE memleak"
 # shellcheck disable=SC2086
-ctest $ARGS || ctest $ARGS --rerun-failed
+if ! ctest $ARGS && [ "$ELEKTRA_AUTO_RERUN" = "1" ]; then
```

Comment:
ELEKTRA_AUTO_RERUN doesn't make much sense, either we rerun or we don't. To add a config option only brings complications (docu, ....)

Reply:
It makes perfect sense IMO, **if** we want to use the automatic rerun in the CI systems. It makes no sense to do an automatic rerun when people run `make run_all` locally. So **if** we want an automatic rerun on CI, I'd add the `ELEKTRA_AUTO_RERUN` env-var to the relevant CI config files, but locally it would still just be a single run.

Especially, when you're working on big changes (like `new-backend` was), it can be very annoying that `make run_all` runs everything twice before it tells you what failed.